app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -865,6 +868,19 @@ watchEffect(() => {
       }),
     ]
     rowData.value = data_.data.map((name) => ({ Value: name }))
+  } else if (isGenericGrid(data_)) {
+    columnDefs.value = data_.headers.map((header) => {
+      if (header.get_child_node_action) {
+        return toLinkField(header.visualization_header, {
+          tooltipValue: header.child_label,
+          headerName: header.visualization_header,
+          getChildAction: header.get_child_node_action,
+        })
+      } else {
+        return toField(header.visualization_header)
+      }
+    })
+    rowData.value = data_.data ? createRowsForTable(data_.data, 0, false) : []
```

Comment:
This condition seems redundant with the type predicate

Reply:
Agreed. Removed it.